File: leetcode/2562.找出数组的串联值.go
func findTheArrayConcVal(nums []int) int64 {
    ans := 0 
    i, j := 0, len(nums) - 1
    for i < j {
        val, _ := strconv.Atoi(strconv.Itoa(nums[i]) + strconv.Itoa(nums[j]))
        ans += val
        i += 1
        j -= 1
    }
    if i == j {
        ans += nums[i]
    }
    return int64(ans)
}

File: leetcode/2562.找出数组的串联值.py
class Solution:
    def findTheArrayConcVal(self, nums: List[int]) -> int:
        n = len(nums)
        i = 0
        j = n-1
        ans = 0
        while i < j:
            ans += int(str(nums[i]) + str(nums[j]))
            i += 1
            j -= 1
        if i == j:
            ans += nums[i]
        return ans